### PR TITLE
usdf: recvv and recvmsg for EP_MSG type

### PR DESCRIPTION
**Pull request by ajaykulk**
Signed-off-by: Ajay Kulkarni <ajaykulk@cisco.com>

@goodell please review

### CODE REVIEW

File: prov/usnic/src/usdf_msg.c [hunk 1]
```diff
@@ -210,7 +210,50 @@ ssize_t
 usdf_msg_recvv(struct fid_ep *fep, const struct iovec *iov, void **desc,
                  size_t count, fi_addr_t src_addr, void *context)
 {
-	return -FI_ENOSYS;
+	struct usdf_ep *ep;
+        struct usdf_rx *rx;
```
Comment (jsquyres) [inline prov/usnic/src/usdf_msg.c]: Watch the inconsistent indenting.  Please use actual tabs, per libfabric style (I think we have some older usnic/usnic_direct code that uses 4-space tabs, but we try to add all new code with libfabric-style actual tabs).

**Comment by shefty** [comment]
Is this updated needed for 1.0?

**Comment by ajaykulk** [comment]
No this is not needed for 1.0.

Thanks,
Ajay

File: prov/usnic/src/usdf_msg.c [hunk 2]
```diff
@@ -466,9 +509,61 @@ usdf_msg_inject(struct fid_ep *fep, const void *buf, size_t len,
...
+	rqe->ms_context = msg->context;
+	tot_len = 0;
+	for (i = 0; i < msg->iov_count; ++i) {
+		rqe->ms_iov[i].iov_base = (void *)iov[i].iov_base;
+		rqe->ms_iov[i].iov_len = iov[i].iov_len;
+		tot_len += iov[i].iov_len;
+	}
+	rqe->ms_last_iov = msg->iov_count - 1;
+
+
```
Comment (goodell) [inline prov/usnic/src/usdf_msg.c]: kill this extra blank line

**Comment by goodell** [comment]
@ajaykulk sorry for the delayed review on this.  The logic all seems OK to me, but the whitespace in the patch needs to be cleaned up ([as Jeff noted](https://github.com/ofiwg/libfabric/pull/998#discussion_r29452673)).  Please re-spin and update the PR.

**Comment by ajaykulk** [comment]
@goodell I have completed the formatting. Please review.

**Comment by goodell** [comment]
:+1:

**Pull request merged**